qiskit/pulse/transforms/alignments.py

```diff
@@ -350,7 +350,7 @@ def pad(schedule: Schedule,
         until: Optional[int] = None,
         inplace: bool = False
         ) -> Schedule:
-    """Pad the input Schedule with ``Delay``s on all unoccupied timeslots until
```

Comment:
I don't see the problem with this comment grammatically, is it a problem with the linter? Could you please reply with a bit of detail below?